core/dive.c

```diff
@@ -36,7 +36,7 @@ const char *cylinderuse_text[] = {
 	QT_TRANSLATE_NOOP("gettextFromC", "OC-gas"), QT_TRANSLATE_NOOP("gettextFromC", "diluent"), QT_TRANSLATE_NOOP("gettextFromC", "oxygen"), QT_TRANSLATE_NOOP("gettextFromC", "not used")
 };
 
-// For user visible text
+// For user visible text but still not translated
```

Comment:
Sure, one can do it that way. But what's wrong with (and I thought we had settled on that) making the _ui() version translated (not only "OC" to "Open Circuit" but to "Offenes System") by replacing the NOOP by translate(), so the translation lookup has to be done only once?

Apart from that aesthetic comment I like the four commits in this PR.